release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -66,9 +66,9 @@ On new clusters, LUKS configuration must use the native Ignition mechanism, as p
 With `bootupd`, {op-system} users now have access to a cross-distribution, system-agnostic OS update tool that manages firmware and boot updates in UEFI and legacy BIOS boot modes that run on modern architectures.
 
 [id="ocp-4-7-rhcos-rhel-8-3-packages"]
-==== {op-system} now supports RHEL 8.3
+==== {op-system} now uses {op-system-base} 8.4
 
-{op-system} is now using Red Hat Enterprise Linux (RHEL) 8.3 packages. {product-title} 4.6 and below will stay with RHEL 8.2 packages. This enables you to have the latest fixes, features, and enhancements, such as NetworkManager features, as well as the latest hardware support and driver updates.
+{op-system} now uses {op-system-base-full} 8.4 packages in {product-title} 4.7.24 and above. {product-title} 4.6 and below will stay with {op-system-base} 8.2 packages. This enables you to have the latest fixes, features, and enhancements, such as NetworkManager features, as well as the latest hardware support and driver updates.
```

Comment:
This is totally just a nitpick, but the phrase "OpenShift Container Platform 4.6 and below will stay with RHEL 8.2 packages." felt a little off to me, due to the verb "stay." My suggestion would be to rephrase it with another verb, but totally just a suggestion.

Reply:
I agree, "stay" is confusing. I've updated based on your suggestion and to be in line with wording we are using in the update to 4.8 RNs (https://github.com/openshift/openshift-docs/pull/35265/files). Thanks!